test/src/unit-concepts.cpp

```diff
@@ -121,7 +121,27 @@ TEST_CASE("concepts")
 
         SECTION("StandardLayoutType")
         {
-            CHECK(std::is_standard_layout<json>::value);
+            // Workaround for MSVC library implementation bug
+            struct can_use_standard_layout_trait
+            {
+                struct A { };
+                struct B { };
+
+                struct C: A, B { };
+
+                struct D: C
+                {
+                    int v;
+                };
+
+                constexpr operator bool() const
+                {
+                    // should return true
+                    return std::is_standard_layout<D>::value;
```

Comment:
Microsoft doesn't consider their implemenation wrong. Their [story](https://developercommunity.visualstudio.com/content/problem/753436/msvs2017-incorrect-size-layout-and-first-data-memb.html) is about empty base optimization, backward binary compatibility and non-standard extenstion `__declspec(empty_bases)` which can be used to achieve standard behavior.